src/ragas/prompt/pydantic_prompt.py

```diff
@@ -410,17 +417,12 @@ async def parse_output_string(
                         prompt_value=prompt_value.to_string(),
                     ),
                     callbacks=retry_cb,
+                    retries_left = retries_left - 1,
                 )
                 retry_rm.on_chain_end({"fixed_output_string": fixed_output_string})
-                return await self.parse_output_string(
```

Comment:
`generate_multiple` calls `parse_output_string()` with the result, we don't need to call it here

Reply:
thanks for catching this 🙌🏽